aspnetcore/blazor/security/passkeys-migration.md

```diff
@@ -0,0 +1,168 @@
+---
+title: Implement passkeys in an existing Blazor Web App
+author: guardrex
+description: Learn how to implement passkeys authentication in an ASP.NET Core Blazor Web App.
+ms.author: wpickett
+ms.custom: mvc
+ms.date: 8/14/2025
+uid: blazor/security/passkeys-migration
+---
+# Implement passkeys in an existing Blazor Web App
+
+This guide explains how to add [passkey support](xref:blazor/security/passkeys) to an existing Blazor Web App that has ASP.NET Core Identity authentication configured.
+
+The guidance in this article relies upon an app that was created with **Individual Accounts** for the app's **Authentication type** or [scaffolding Identity into an existing app](xref:security/authentication/scaffold-identity#scaffold-identity-into-a-blazor-project).
+
+## Prerequisites
+
+<!-- UPDATE 10.0 - Remove preview link in favor of the download link ...
+
+* [.NET SDK](https://dotnet.microsoft.com/download) (.NET 10 or later)
+
+-->
+
+* An existing Blazor Web App with ASP.NET Core Identity
+* [.NET 10 SDK](https://dotnet.microsoft.com/download/dotnet/10.0)
+
+## Reference source guidance
+
+The links in this article to .NET reference source load the repository's default branch, which represents the current development for the next release of .NET. To select a tag for a specific release, use the **Switch branches or tags** dropdown list. For more information, see [How to select a version tag of ASP.NET Core source code (dotnet/AspNetCore.Docs #26205)](https://github.com/dotnet/AspNetCore.Docs/discussions/26205).
+
+## Step 1: Update to .NET 10
+
+Update the app to .NET 10 or later. For more information, see <xref:migration/index>.
+
+## Step 2: Update Identity schema version
+
+In `Program.cs`, update the Identity configuration to use schema version 3, which includes passkey support:
+
+```csharp
+builder.Services.AddIdentityCore<ApplicationUser>(options =>
+{
+    options.SignIn.RequireConfirmedAccount = true;
+    options.Stores.SchemaVersion = IdentitySchemaVersions.Version3;
+})
+.AddEntityFrameworkStores<ApplicationDbContext>()
+.AddSignInManager()
+.AddDefaultTokenProviders();
+```
+
+# [Visual Studio](#tab/visual-studio)
```

Comment:
Maybe add another step heading here to clarify that you need to create and run a migration to update the database?